src/components/dashboard/navigation.js

```diff
@@ -38,16 +90,115 @@ class Navigation extends Component {
     this.props.recommendationAction(id)
   }
 
+  
   render() {
+    const {setState} = this.props
+    const genresArray = ['Documentary','Drama','Horror','Mystery','Adventure','History','Musical','Action','Comedy','Thriller','Crime','Romance','Biography', 'Music', '\\N','Family', 'Sci-Fi', 'Fantasy','Western', 'Sport','Animation','War','Adult','Reality-TV', 'News', 'Talk-Show','Game-Show','Film-Noir']
```

Comment:
Is this a comprehensive list or would it need to be updated in the future? If so, it could live in your config for easy updates.